.github/ISSUE_TEMPLATE/propose-an-idea.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Propose an Idea
+about: Used to document discussion around proposed ideas when deciding whether to
+  approve them for the roadmap.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Target Audience**
+The primary audience for this request is the SaaS Governance team that manually receives SBOMs. In addition, other teams (including Harbor developers) will also be uploading and viewing SBOMs.
```

Comment:
This static content shouldn't be in here.

please see https://github.com/CMS-Enterprise/sbom-harbor-ui/pull/111#pullrequestreview-1543702176 for reference.